src/MSBuild/TerminalLogger/TestSummary.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Build.Logging.TerminalLogger
+{
+    internal class TestSummary
```

Comment:
NIT: We are trying to minimize setters abundance. Please consider using `readonly record struct TestSummary(int Totall, ...);`

Reply:
done